### PR TITLE
Add Zora to Tokens Tables

### PR DESCRIPTION
**Pull request by MSilb7**
Adds Zora to ERC20/NFT tokens tables - based on a copy of Base.

### CODE REVIEW

File: models/tokens/tokens_erc20.sql [hunk 1]
```diff
@@ -27,4 +27,6 @@ SELECT 'base' as blockchain, contract_address, symbol, decimals FROM {{ ref('tok
 UNION ALL
 SELECT 'celo' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_celo_erc20') }}
 UNION ALL
-SELECT 'zksync' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_zksync_erc20') }}
\ No newline at end of file
+SELECT 'zksync' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_zksync_erc20') }}
+UNION ALL
+SELECT 'zksync' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_zora_erc20') }}
```
Comment (jeff-dude) [inline models/tokens/tokens_erc20.sql:32]: update blockchain to zora plz

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/tokens/tokens_erc20.sql (hunk 1 above)
Comment (MSilb7) [reply models/tokens/tokens_erc20.sql:32]: lol whoops

**Comment by MSilb7** [comment]
resolved @jeff-dude

**Review by jeff-dude — APPROVED** [review]
thank you ser 🤝

**Pull request merged**